server/server.go

```diff
@@ -29,20 +29,20 @@ const (
 
 // Server implements the RuntimeService and ImageService
 type Server struct {
-	config       Config
-	runtime      *oci.Runtime
-	store        sstorage.Store
-	images       storage.ImageServer
-	storage      storage.RuntimeServer
-	stateLock    sync.Mutex
-	updateLock   sync.RWMutex
-	state        *serverState
-	netPlugin    ocicni.CNIPlugin
-	podNameIndex *registrar.Registrar
-	podIDIndex   *truncindex.TruncIndex
-	ctrNameIndex *registrar.Registrar
-	ctrIDIndex   *truncindex.TruncIndex
-	imageContext *types.SystemContext
+	config        Config
+	runtime       *oci.Runtime
+	store         sstorage.Store
+	imageServer   storage.ImageServer
+	runtimeServer storage.RuntimeServer
```

Comment:
We also have a runtime field. Should we call these storageRuntimeService and storageImageService?